packages/bundlemon/src/main/outputs/outputs/json.ts

```diff
@@ -0,0 +1,53 @@
+import * as yup from 'yup';
+import fs from 'fs';
+import { Report } from 'bundlemon-utils';
+import { createLogger } from '../../../common/logger';
+import { validateYup } from '../../utils/validationUtils';
+import type { Output, OutputInstance } from '../types';
+
+const NAME = 'json';
+const DEFAULT_FILENAME = 'bundlemon-results.json';
+
+const logger = createLogger(`${NAME} output`);
+
+interface JsonOutputOptions {
+  fileName: string;
+}
+
+function validateOptions(options: unknown): JsonOutputOptions | undefined {
+  const schema = yup
```

Comment:
I see that there is a typescript error here.
I pushed a fix to master, merge It to your branch and change this line to be like [here](https://github.com/LironEr/bundlemon/pull/63/files#diff-37377742105178e9fb9cc246e5002f3b0adb4877a2da2a624996a47de5855fe4R20), should fix the problem.
```suggestion
  const schema: yup.SchemaOf<JsonOutputOptions, JsonOutputOptions> = yup
```

Reply:
will do!

Reply:
@LironEr  I rebased with master after fetching your recent canges, then applied your suggested change, but now I'm getting such error: 
![Screenshot from 2021-11-02 20-07-36](https://user-images.githubusercontent.com/58426925/139929465-a40c4a7a-f6a1-456c-83d2-8e3716f58172.png)

Any idea how how to solve it?
This is how I installed the project on my machines:
```sh
git clone bla-bla-bla
cd bundlemon
npm i -g yarn
yarn install
```
then I try to build with
```sh
npm run build-packages
```
or
```sh
cd packages/bundlemon
npm run build
```

Reply:
TypeScript version used 
![Screenshot from 2021-11-02 20-20-28](https://user-images.githubusercontent.com/58426925/139932016-3442ea0a-ed01-4359-98cf-68be77499565.png)

Reply:
Done in https://github.com/LironEr/bundlemon/pull/61/commits/a008e704eae948445003939ffdd77d8f34bbecbf